test/tests.bats

```diff
@@ -929,8 +929,9 @@ EOF
   assert_output 1
 }
 
-
+# TODO investigate why these commands fail
```

Comment:
While I think it's better visibility having the TODO above the test case here, it is inconsistent with the other two that are placed before the `skip` line instead.